tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -718,6 +722,12 @@ where
 				)));
 				Ok(())
 			},
+			TrustedCall::set_scheduled_mrenclave(_, bn, mrenclave) => {
+				GLOBAL_SCHEDULED_ENCLAVE
+					.update(bn, mrenclave)
+					.map_err(|_| StfError::SetScheduledMrEnclaveFailed)?;
```

Comment:
Here we lose the original root cause - maybe simply log it?

Reply:
done